test/evaluation/limitDisclosureEvaluationHandler.spec.ts

```diff
@@ -2,34 +2,117 @@ import fs from 'fs';
     const pdSchema: PresentationDefinition = getFile(
       './test/dif_pe_examples/pd/pd-simple-schema-age-predicate.json'
     ).presentation_definition;
     const vpSimple: VerifiablePresentation = getFile('./test/dif_pe_examples/vp/vp-simple-age-predicate.json');
     const evaluationClient: EvaluationClient = new EvaluationClient();
-    evaluationClient.evaluate(pdSchema, vpSimple.verifiableCredential, HOLDER_DID);
-    expect(evaluationClient.verifiableCredential[0].credentialSubject['etc']).toEqual(undefined);
+    evaluationClient.evaluate(pdSchema, vpSimple.verifiableCredential, HOLDER_DID, LIMIT_DISCLOSURE_SIGNATURE_SUITES);
+    expect(evaluationClient.verifiableCredential[0].credentialSubject['etc']).toBeUndefined();
   });
 
-  it("should return ok if verifiable Credential doesn't have the birthPlace field", function () {
+  it("should return ok if verifiable Credential doesn't have the birthPlace field", () => {
     const pdSchema: PresentationDefinition = getFile(
       './test/dif_pe_examples/pd/pd-schema-multiple-constraints.json'
     ).presentation_definition;
     const vpSimple: VerifiablePresentation = getFile('./test/dif_pe_examples/vp/vp-multiple-constraints.json');
     pdSchema.input_descriptors[0].schema.push({ uri: 'https://www.w3.org/2018/credentials/v1' });
     const evaluationClient: EvaluationClient = new EvaluationClient();
-    evaluationClient.evaluate(pdSchema, vpSimple.verifiableCredential, HOLDER_DID);
-    expect(evaluationClient.verifiableCredential[0]['birthPlace']).toEqual(undefined);
+    evaluationClient.evaluate(pdSchema, vpSimple.verifiableCredential, HOLDER_DID, LIMIT_DISCLOSURE_SIGNATURE_SUITES);
+    expect(evaluationClient.verifiableCredential[0].credentialSubject['birthPlace']).toBeUndefined();
+  });
+
+  it('limit disclosure should not be supported', () => {
+    const pdSchema: PresentationDefinition = getFile(
+      './test/dif_pe_examples/pd/pd-schema-multiple-constraints.json'
+    ).presentation_definition;
+    const vpSimple: VerifiablePresentation = getFile('./test/dif_pe_examples/vp/vp-multiple-constraints.json');
+    pdSchema.input_descriptors[0].schema.push({ uri: 'https://www.w3.org/2018/credentials/v1' });
+    vpSimple.verifiableCredential[0].proof.type = 'limit disclosure unsupported';
+    const evaluationClient: EvaluationClient = new EvaluationClient();
+    evaluationClient.evaluate(pdSchema, vpSimple.verifiableCredential, HOLDER_DID, LIMIT_DISCLOSURE_SIGNATURE_SUITES);
+    expect(evaluationClient.verifiableCredential[0].credentialSubject['birthPlace']).toEqual('Maarssen');
+    expect(evaluationClient.results[7]).toEqual({
+      evaluator: 'LimitDisclosureEvaluation',
+      input_descriptor_path: '$.input_descriptors[0]',
+      message: 'Limit disclosure not supported',
+      status: 'error',
+      verifiable_credential_path: '$[0]',
+    });
+  });
+
+  it('limit disclosure should not be supported', () => {
```

Comment:
If manddatory fields are absent the error should be reported

 is a better title.

Reply:
Same as above for that one, what about: 'should report an error if mandatory fields are absent'?

Reply:
Looks good.